lib/utils/moment/src/index.js

```diff
@@ -24,4 +24,25 @@ moment.toYYYYMM = (period) => {
   return time.getUTCFullYear() * 100 + time.getUTCMonth() + 1;
 };
 
+//
+// Returns a timestamp (in milliseconds since epoch), at the specified
+// year, month, day, hour, minute, second, and milliseconds.
+//
+moment.utcTimestamp = (year, month, day, hour, minute, second, millis) => {
```

Comment:
Why is `moment.utcTimestamp` shorter or more expressive than `moment.utc([...]).valueOf()` ?

Reply:
After discussion we decided to keep the new utcTimestamp function.